### PR TITLE
add backchannel logout url attribute

### PR DESCRIPTION
**Pull request by wolfaba**

### CODE REVIEW

**Comment by treydock** [comment]
Be good to add unit and acceptance tests:

Unit:

https://github.com/treydock/puppet-module-keycloak/blob/master/spec/unit/puppet/type/keycloak_client_spec.rb#L80

Acceptance:

https://github.com/treydock/puppet-module-keycloak/blob/master/spec/acceptance/5_client_spec.rb

**Comment by wolfaba** [comment]
dear @treydock 
I hope this is, what you wanted and I hope it's correct.
I very sorry if it's not correct, but I am just sysadmin, neither ruby nor puppet module developer. I was able to add the backchannel logout URL only because the module is written really good and even that I'm not profi, I have understood (hopefully) the functionality and could extend the code.
Thank you.
Regards.
Robert.

File: lib/puppet/type/keycloak_client.rb [hunk 1]
```diff
@@ -158,6 +158,10 @@ def insync?(is)
     desc 'rootUrl'
   end
 
+  newproperty(:backchannel_logout_url) do
+    desc 'backchannel.logout.url'
+  end 
+
```
Comment (treydock) [inline lib/puppet/type/keycloak_client.rb:164]:
```suggestion
  end

```

The validation tests are failing because of trailing white space

Comment (wolfaba) [reply lib/puppet/type/keycloak_client.rb]: fixed, thank you

**Comment by treydock** [comment]
This will be released with v7.14.0.  The forge release will be present once Github Actions complete

**Comment by wolfaba** [comment]
thank you

**Pull request merged**